charts/kueue/templates/webhook/webhook.yaml

```diff
@@ -0,0 +1,234 @@
+{{- if .Values.enableWebhook }}
```

Comment:
why would we give this option? disabling validation and defaulting could cause many problems.

Reply:
I added the option to enable/disable webhooks in the kustomization based on the design of [kustomization](https://github.com/kubernetes-sigs/kueue/blob/main/config/default/kustomization.yaml).
However, as you pointed out, the components/webhook is enabled in the kustomization file.
I will remove this option from all files.

Reply:
Removed enableWebhook.